lib/stdlib/select.juttle

```diff
@@ -59,3 +59,28 @@ export sub percentile(field, p, by=[]) {
 export sub median(field, by=[]) {
     percentile -field field -p 0.5 -by by
 }
+
+//
+// forward the top N points ordered by the `by` field provided
+//
+// Parameters:
+//
+//   * n: number of points to forward
+//   * by: (opt) field to sort by
```

Comment:
This comment implies `by` is optional but it's not, is it? I don't see a default